docs/llm_compressor.qmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "LLMCompressor Sparse Fine-tuning"
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    number-sections: true
+execute:
+  enabled: false
+---
+
+# LLMCompressor Integration
+
+Fine-tune sparsified models in Axolotl using [LLMCompressor](https://github.com/vllm-project/llm-compressor).
+
+This integration enables fine-tuning of models **already sparsified** using LLMCompressor. 
```

Comment:
Nit: sounds like they have to have been made sparse through llmcompressor, but can still run even if made sparse through something else.

Reply:
Called it out explicitly in integration README now!